app/Http/Controllers/api/RecruiterController.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Exceptions\UserNotAuthenticatedException;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\UpdateRecruiterRequest;
+use App\Http\Requests\UserRequest;
+use App\Http\Resources\RecruiterListResource;
+use App\Http\Resources\RecruiterResource;
+use App\Models\Recruiter;
+use App\Models\User;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+
+class RecruiterController extends Controller
+{
+    public function index()
+    {
+        try {
+            $recruitersList = Recruiter::all();
+
+            if ($recruitersList->isEmpty()) {
+
+                throw new ModelNotFoundException(__('No hi ha reclutadors a la base de dades.'), 404);
```

Comment:
Who should be responsible of "painting things pretty" to the final User, the backend or the frontend?

Reply:
The responsibility would be on the front end, to present the data in an attractive way.

Reply:
The responsibility would be on the front end, to present the data in an attractive way.